frontend/packages/ux-editor/src/components/config/Expressions/DataSourceValue.tsx

```diff
@@ -82,15 +83,16 @@ export const DataSourceValue = ({
       );
     case DataSource.String:
       return (
-        <LegacyTextField
+        <Textfield
           onChange={(e) => specifyDataSourceValue(e.target.value, isComparableValue)}
           value={currentValue as string}
         />
       );
     case DataSource.Number:
       return (
-        <LegacyTextField
-          formatting={{ number: {} }}
+        <Textfield
+          data-testid={testids.textboxNumber}
```

Comment:
we should not need to use test-id on this input field. Input fields without a label are not accessible. Since we do not have any number validation we should revert these changes.

We need to get our new studio-ui where we can add our TextField which supports number validation before we can change TextFields for numbers.

Reply:
I used it for test after i tried all the queries from priority, but they did not work, I removed it :blush: